configuration/pih/htmlforms/section-obgyn-initial.xml

```diff
@@ -157,6 +157,8 @@
     const obGynEncounterType= 'd83e98fd-dc7b-420f-aa3f-36f648b4483d';
     const typeOfVisit = '164181AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA';
     const intakeVisit = '164180AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA';
+    const whProcedurePerformed="1651AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA";
+    const whProcedures="57755670-c183-494b-bd5e-723e0978c3e3";
```

Comment:
I like the list AND adding dates is great.  Shouldn't this be the other convset?

const whSterilizationProcedures = "46cbbdd9-9cc2-42c6-8ecb-4dbe6ea01354";

![image](https://user-images.githubusercontent.com/1164849/183124267-6f7ca5d3-7a4a-498e-b3aa-d03cc6ec3dad.png)

Reply:
AND nice work @cioan !